pennylane_lightning/src/rework/Apply.cpp

```diff
@@ -63,7 +36,7 @@ void Pennylane::constructAndApplyOperation(
     
     vector<size_t> internalIndices = generateBitPatterns(opWires, qubits);
 
-    vector<unsigned int> externalWires = getIndicesAfterExclusion(opWires, qubits);
+    vector<unsigned int> externalWires = Pennylane::getIndicesAfterExclusion(opWires, qubits);
```

Comment:
Was is it required to add the `Pennylane::` scope here?

Reply:
Nope, removed.